app/controllers/api/floating_ips_controller.rb

```diff
@@ -1,4 +1,40 @@
 module Api
   class FloatingIpsController < BaseController
+    def create_resource(_type, _id = nil, data = {})
```

Comment:
Does this come from a PUT or from a POST with "create" action (or both)?  What does the incoming payload look like and does it match the shape of the GET request?